src/page-storage/auto-tagging.js

```diff
@@ -0,0 +1,47 @@
+import compromise from 'compromise'
+import index from 'src/search/search-index'
+import db from 'src/pouchdb'
+
+const idb = index.db._db
```

Comment:
Probably won't matter much longer as we're moving to a new index soon but just a YSK: as a bit of a convention due to lack of straightforward encapsulation in JS ([soon, maybe](https://tc39.github.io/proposal-class-fields/)), properties starting with `_` shouldn't be accessed directly. Think of it like in leading underscores in PEP-8, but I think python actually excludes underscore props when importing modules